brave/brave-core
# Fix renderer crash from adblock page(removed stats from adblock webui)

fix https://github.com/brave/brave-browser/issues/13775

When navigting from adblock to other pages, renderer is crashed.
This crash comes from referring RFH to try update stats whlie
new naviagion occurs. 
As we don't need to show stats, stats updating logic is removed.

![image](https://user-images.githubusercontent.com/6786187/106836667-effb6380-66dc-11eb-992a-cab6d8b944c8.png)



## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.

## Test Plan:
1. Load any sites in advance
2. Load brave://adblock in the same tab
3. Navigate back to previous or type www.brave.com and check renderer is not crashed.
